### ISSUE TITLE
HolographicDisplays compatability

### ISSUE THREAD

**Issue by csorrows** [NLthijs48/AreaShop#87]
HolographicDisplays option instead of using signs for rent and buy regions.

**Comment by EvilOlaf** [comment]
I prefer the signs. I like the idea though.

**Comment by NLthijs48** [comment]
I agree this would be cool to add, will probably write some abstraction to display region data to support this, as I can imagine there are more ways to display it.

**Comment by csorrows** [comment]
I have been thinking about this one and how it should be implemented. Here is what I have come up with.

Creation method 1: Use the same function as you use when replacing/adding signs. As when you put [as] on the first line and region on the second line. Create a command to select the pre-made hologram and hook it into changing the text in the hologram just as it would when creating a sign.

Creation method 2: Create the normal sign but with [hrent] or [hsell] on the top line. When complete, the sign would be replaced by a hologram displaying the same info as entered on the sign.

Usage: It would hook into the hologram's clicks so the function of the hologram would work the same as the sign.

I'm not saying these are the best or only aspects the holograms would have, just what I have thought of.
